frontend/src/components/IstioStatus/IstioStatus.tsx

```diff
@@ -107,8 +107,10 @@ export const meshLinkStyle = kialiStyle({
 
 export const IstioStatusComponent: React.FC<Props> = (props: Props) => {
   const { t } = useKialiTranslation();
+  const location = useLocation();
```

Comment:
This `location` variable name makes sense but there is also a property called `props.location`. To be honest, I don't think we even need `props.location`. You could probably remove it completely. If nothing else I think you should rename it.